exercises/transpose/test/Tests.hs

```diff
@@ -0,0 +1,181 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import Transpose (transpose)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "responseFor" $ for_ cases test
```

Comment:
`describe` should describe what it's testing, but I think we are not testing `responseFor` here, so let's change this